torch/_utils.py

```diff
@@ -196,7 +196,13 @@ def set_tensor_metadata(tensor, metadata):
 
 
 def _rebuild_tensor_v2(
-    storage, storage_offset, size, stride, requires_grad, backward_hooks, metadata=None
+    storage,
```

Comment:
unrelated lint?

Reply:
was leftover from when I did modify this file. Undone now